proposals/20220919-airflow_orchestration.md

```diff
@@ -0,0 +1,35 @@
+#### SIG TFX-Addons
+# Project Proposal
+
+**Your name:** Varun Murthy
+
+**Your email:** murthyvs@google.com
+
+**Your company/organization:** Google Core ML TFX Team (MTV + Seoul)
+
+**Project name:** Airflow Orchestration
+
+## Project Description
+Moving Airflow to TFX add-ons due to decreasing native support.
+
+## Project Category
+Orchestrator
+
+## Project Use-Case(s)
+We are moving Airflow from tfx/orchestration to tfx-addons. Native support for Airflow won't be provided in the near future.
+
+## Project Implementation
+Move Airflow from tfx/orchestration to tfx-addons.
+
+## Project Dependencies
+Same as tfx/orchestration/airflow.
+
+## Project Team
+Google Core ML TFX Team (MTV + Seoul)
```

Comment:
The project team members need to be listed, including Github IDs and email addresses.  The team members themselves will be responsible for maintaining the project, so they need to be aware of their responsibilities.